.github/workflows/devcontainer.yml

```diff
@@ -43,11 +37,12 @@ jobs:
       - run: |
           # Strip jsonc comments because `jq` doesn't support them.
           image=$(sed -E '/^\s*\/\/.*/d' .devcontainer/devcontainer.json |jq -Mr .image)
+          line=$(grep -nF '"image"' .devcontainer/devcontainer.json |cut -d':' -f1 |head -n1)
           if [ "$image" == "null" ]; then
-            echo "::error file=.devcontainer/devcontainer.json::Missing image"
+            echo "::error file=.devcontainer/devcontainer.json,line=$line::Missing image"
             exit 1
           fi
           if ! docker pull "$image" ; then
-            echo "::error file=.devcontainer/devcontainer.json::Unable to pull image: $image"
+            echo "::error file=.devcontainer/devcontainer.json,line=$line::Unable to pull image: $image"
             exit 1
           fi
```

Comment:
it seems like a potential follow-up might be to move these actions to the `dev` repo, as well, so that multiple repos can depend on them from a single source of truth?

Reply:
Yes, we may want to standardize our actions; but the dev repo probably isn't the right place for them.